src/specs/erc4626/readme.md

```diff
@@ -0,0 +1,35 @@
+# Spec for ERC4626 Bridge
+
+## What does this bridge do? Why did you build it?
+
+The bridge allows people to issue and redeem shares of any ERC4626 vault after the vault is listed in the contract.
+
+## What protocol(s) does the bridge interact with?
+
+Any ERC4626 compatible vault and the underlying assets.
+
+## What is the flow of the bridge?
+
+The bridge only support 2 flows:
+
+1. ERC4626 share issuance (this flow gets triggered when `_auxData` = 0),
+2. ERC4626 share redemption (this flow gets triggered when `_auxData` = 1).
+
+## Please list any edge cases that may restrict the usefulness of the bridge or that the bridge explicitly prevents.
+
+Vault has to be listed first before using the bridge with it.
+
+## How can the accounting of the bridge be impacted by interactions performed by other parties than the bridge? Example, if borrowing, how does it handle liquidations etc.
+
```

Comment:
There should probably be a note that as the underlying ERC4626, might be done with a proxy or admin rights, it could be possible to alter the balances, which could mess up the accounting inside the rollup processor if some of the tokens are burned etc.

Reply:
Addressed in b5dc4e5ae99482a2f47ff40e154c08f3d0fdbe8a